.bazelrc

```diff
@@ -1,2 +1,2 @@
 build --workspace_status_command hack/build/print-workspace-status.sh
-test --test_output=errors --test_output=streamed
+test --test_output=streamed
```

Comment:
This will make the unit tests run single threaded. Can we have a dev bazel profile?

Reply:
I'll just make it `errors`, I don't think there's any compelling reason for it to be `streamed`. Even running locally I find the output from `errors` to be sufficient.